pkg/transaction/transaction.go

```diff
@@ -53,7 +54,7 @@ type Interface interface {
 }
 
 type UpgradeHelper interface {
-	SyncImageContent(*deployment.ImageSource, *Transaction) error
+	SyncImageContent(*deployment.ImageSource, *Transaction, ...unpack.Opt) error
```

Comment:
Can these options be part of the struct itself in order not to modify the interface signature?

Reply:
Actually, it looks like it's a limitation of the `Init()` function of the other interface. Perhaps we could improve this in the future. 🤔

Reply:
Yep, this code is a bit convoluted and I will try to revisit this in the coming months!